src/components/InputNewTodo/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './InputNewTodo.module.css'
+
+type InputNewTodoProps = {
+    todoTitle: string,
+    onChange: (todoTitle: string) => void,
+    onSubmit: (todo: any) => void,
+
+}
+type InputNewTodoState = {
+    value: string
+}
+
+export class InputNewTodo extends React.Component<InputNewTodoProps, InputNewTodoState> {
+    componentDidUpdate(prevProps: Readonly<InputNewTodoProps>, prevState: Readonly<InputNewTodoState>, snapshot?: any) {
```

Comment:
this line should be removed. This state shouldn't be stored outside. just pass this value state when you submit